tests/CatalogTests/Registration/RegistrationMakerCatalogItemTests.cs

```diff
@@ -265,67 +272,77 @@ public static IEnumerable<object[]> CreatePageContent_SetsDeprecationInformation
                     _graph,
                     _registrationBaseAddress,
                     isExistingItem: false,
+                    postProcessGraph: shouldPostProcess ? RegistrationCollector.FilterOutDeprecationInformation : g => g,
                     packageContentBaseAddress: _packageContentBaseAddress)
                 {
                     BaseAddress = _baseAddress,
                 };
+
                 RegistrationMakerCatalogItem.PackagePathProvider = new PackagesFolderPackagePathProvider();
                 var context = new CatalogContext();
 
                 // Act
                 var content = item.CreatePageContent(context);
 
                 // Assert
-                var deprecationObjectNode = _graph
+                var deprecationPredicateTriples = content
                     .GetTriplesWithSubjectPredicate(
-                        _graph.CreateUriNode(_catalogUri), 
-                        _graph.CreateUriNode(Schema.Predicates.Deprecation))
+                        content.CreateUriNode(_catalogUri),
+                        content.CreateUriNode(Schema.Predicates.Deprecation));
+
+                if (shouldPostProcess)
+                {
+                    Assert.Empty(deprecationPredicateTriples);
+                    return;
+                }
+
+                var deprecationObjectNode = deprecationPredicateTriples
                     .Single()
                     .Object;
 
-                var deprecationTriples = _graph.GetTriplesWithSubject(deprecationObjectNode);
+                var deprecationTriples = content.GetTriplesWithSubject(deprecationObjectNode);
```

Comment:
This test was silently broken before: we were testing `_graph`, the input to the `RegistrationMakerCatalogItem`, instead of `content`, which is the `RegistrationMakerCatalogItem`'s output.